src/frontends/tensorflow/src/op/assert.cpp

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <numeric>
+
+#include "op_table.hpp"
+#include "openvino/core/validation_util.hpp"
+
+using namespace std;
+
+namespace ov {
+namespace frontend {
+namespace tensorflow {
+namespace op {
+
+OutputVector translate_assert_op(const NodeContext& node) {
+    default_op_checks(node, 1, {"Assert"});
+    auto cond = node.get_input(0);
+    auto cond_const = get_constant_from_source(cond);
+    TENSORFLOW_OP_VALIDATION(node,
+                             cond_const,
+                             "[TensorFlow Frontend] The condition must be constant for further model conversion.");
```

Comment:
This `Tensorflow Frontend` tag seems weird, why can't it be included inside the `TENSORFLOW_OP_VALIDATION` macro?

Reply:
I previously used this tag already but I like you suggestion to inject some TF FE specific tag into `TENSORFLOW_OP_VALIDATION`. Looks better to do this in the separate PR for cleaning-up TF FE messages, for example.